PizzaPyWebApp/templates/partials/events/hero.html

```diff
@@ -0,0 +1,128 @@
+<div class="relative flex flex-col items-center justify-center w-full">
+    <img src="../../static/images/python-3d.png" alt="PizzaPy Logo"
+        class="absolute scale-[1.5] md:scale-[2.0] lg:scale-[3.5] rotate-[11deg] opacity-30 blur-xl -z-50 lg:right-[20%] lg:bottom-[50%]" />
+    <div class="flex flex-col items-center w-full my-10 lg:my-20">
```

Comment:
Looks too tight on mobile devices. Perhaps add some padding?

On that note, line height is too spaced out. Would making it tighter improve it?

![image](https://github.com/pizzapy/pizzapy-website/assets/105530193/584e15c3-88e8-4aaf-8b44-6dcc848f9f7f)

Reply:
https://github.com/pizzapy/pizzapy-website/assets/42131558/2bb8025f-0eac-4570-aae2-69312d5c084d